### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by AmedeeBulle**
- Update README files to reflect new repo name (except for the projects having a running PR #265, #270)
- Basic markdown linting

--
Signed-off-by: Philippe Vanhaesendonck <philippe.vanhaesendonck@oracle.com>

### CODE REVIEW

File: ContainerRegistry/README.md [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
+
 This Vagrant project will provision an Oracle Linux 7 VM running a local
 Container Registry.
 
 It can be used as cache for the Oracle Container Registry, in particular for
 the Kubernetes containers.
 
 ## Prerequisites
+
 1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM
```
Comment (totalamateurhour) [inline ContainerRegistry/README.md:11]:
```suggestion
1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up Vagrant with either VirtualBox or KVM
```

File: DockerEngine/README.md [hunk 2]
```diff
@@ -1,27 +1,32 @@
-# Vagrantfile to set up Oracle Linux 7 with Docker engine
+# Vagrant project to set up Oracle Linux 7 with Docker engine
+
 A Vagrantfile that installs and configures Docker engine on Oracle Linux 7 with Btrfs as storage
 
 ## Prerequisites
-1. Install [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-2. Install [Vagrant](https://vagrantup.com/)
+
+Read the [prerequisites in the top level README](../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM
```
Comment (totalamateurhour) [inline DockerEngine/README.md:7]:
```suggestion
Read the [prerequisites in the top level README](../README.md#prerequisites) to set up Vagrant with either VirtualBox or KVM
```

File: Kubernetes/README.md [hunk 3]
```diff
@@ -1,34 +1,38 @@
...
-__Note:__ this Vagrantfile is deprecated in favour of the [OLCNE](../OLCNE)
+__Note:__ this Vagrant project is deprecated in favour of the [OLCNE](../OLCNE)
 one. It can still be used to provision earlier version of Kubernetes clusters
 (e.g. 1.12).
 
 ## Prerequisites
-1. Install [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-1. Install [Vagrant](https://vagrantup.com/)
+
+Read the [prerequisites in the top level README](../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM
```
Comment (totalamateurhour) [inline Kubernetes/README.md:12]:
```suggestion
Read the [prerequisites in the top level README](../README.md#prerequisites) to set up Vagrant with either VirtualBox or KVM
```

**Comment by Djelibeybi** [comment]
We only need a single `LICENSE` file in the root of the repo. All the rest can be removed.

File: Kubernetes/README.md [hunk 4]
```diff
@@ -1,34 +1,38 @@
-# Vagrantfile to setup a Kubernetes Cluster on Oracle Linux 7
-This Vagrantfile will provision a Kubernetes cluster with one master and _n_
+# Vagrant project to setup a Kubernetes Cluster on Oracle Linux 7
+
+This Vagrant project will provision a Kubernetes cluster with one master and _n_
```
Comment (totalamateurhour) [inline Kubernetes/README.md:3]: Recommend we stick with referring to "master" nodes until Kubernetes upstream changes this terminology.

File: LAMP/README.md [hunk 5]
```diff
@@ -1,39 +1,42 @@
 # ol7-lamp
-A vagrant box that provisions Oracle Linux LAMP solution automatically, using Vagrant, an Oracle Linux 7 box and a shell script.
+
+A Vagrant project that provisions Oracle Linux LAMP solution automatically, using Vagrant, an Oracle Linux 7 box and a shell script.
 
 ## Prerequisites
-1. Install [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-2. Install [Vagrant](https://vagrantup.com/)
+
+Read the [prerequisites in the top level README](../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM
```
Comment (totalamateurhour) [inline LAMP/README.md:7]:
```suggestion
Read the [prerequisites in the top level README](../README.md#prerequisites) to set up Vagrant with either VirtualBox or KVM
```

File: OLCNE/README.md [hunk 6]
```diff
@@ -50,19 +53,22 @@ the `vagrant` user). E.g.:
...
+
 The Vagrantfile can be used _as-is_; there are a couple of parameters you
 can set to tailor the installation to your needs.
 
 ### How to configure
+
 There are several ways to set parameters:
+
 1. Update the Vagrantfile. This is straightforward; the downside is that you
 will loose changes when you update this repository.
```
Comment (totalamateurhour) [inline OLCNE/README.md:69]:
```suggestion
will lose changes when you update this repository.
```

File: Kubernetes/README.md [hunk 7]
```diff
@@ -55,15 +59,18 @@ done by the `kubeadm-setup-master.sh` and `kubeadm-setup-worker.sh` helper
...
+
 The Vagrantfile can be used _as-is_; there are a couple of parameters you
 can set to tailor the installation to your needs.
 
 ### How to configure
+
 There are several ways to set parameters:
+
 1. Update the Vagrantfile. This is straightforward; the downside is that you
 will loose changes when you update this repository.
```
Comment (totalamateurhour) [inline Kubernetes/README.md:71]:
```suggestion
will lose changes when you update this repository.
```

File: OracleDG/README.md [hunk 8]
```diff
@@ -8,25 +8,13 @@ two Oracle RDBMS (18c, 19c) hosts configured with Oracle Data Guard, using Vagra
...
-  - `yum install -y ruby-devel libvirt-devel`
-4. Install vagrant-libvirt as user
-  - `vagrant plugin install vagrant-libvirt`
-5. You need to download Database binary separately
-
-#### Note: *Using KVM/libVirt provider you may need to disable or manage host firewall to permit NFS traffic with the guest VMs*
-#### Note: if you are going to use KVM on OL7/OL8 please read 'https://blogs.oracle.com/linux/getting-started-with-the-vagrant-libvirt-provider-for-oracle-linux'
+## Prerequisites
 
+1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM
```
Comment (totalamateurhour) [inline OracleDG/README.md:13]:
```suggestion
1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up Vagrant with either VirtualBox or KVM
```

File: OracleFPP/README.md [hunk 9]
```diff
@@ -1,30 +1,18 @@
...
-  - `yum install -y ruby-devel libvirt-devel`
-4. Install vagrant-libvirt as user
-  - `vagrant plugin install vagrant-libvirt`
-5. You need to download Database binary separately
-
-#### Note: *Using KVM/libVirt provider you may need to disable or manage host firewall to permit NFS traffic with the guest VMs*
-#### Note: if you are going to use KVM on OL7/OL8 please read 'https://blogs.oracle.com/linux/getting-started-with-the-vagrant-libvirt-provider-for-oracle-linux'
+## Prerequisites
 
+1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM
```
Comment (totalamateurhour) [inline OracleFPP/README.md:11]:
```suggestion
1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up Vagrant with either VirtualBox or KVM
```

File: OracleLinux/6/README.md [hunk 10]
```diff
@@ -1,36 +1,40 @@
 # ol6-vagrant
-A vagrant box that provisions Oracle Linux automatically, using Vagrant, an Oracle Linux 6 box and a shell script.
+
+A Vagrant project that provisions Oracle Linux automatically, using Vagrant, an Oracle Linux 6 box and a shell script.
 
 ## Prerequisites
-1. Install [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-2. Install [Vagrant](https://vagrantup.com/)
+
+Read the [prerequisites in the top level README](../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM
```
Comment (totalamateurhour) [inline OracleLinux/6/README.md:7]:
```suggestion
Read the [prerequisites in the top level README](../README.md#prerequisites) to set up Vagrant with either VirtualBox or KVM
```

File: OracleRAC/README.md [hunk 11]
```diff
@@ -256,12 +257,9 @@ To run scripts in a specific order, prefix the file names with a number, e.g., `
         </ip>
       </network>
 - If you are behing a proxy, set the following env variables
```
Comment (totalamateurhour) [inline OracleRAC/README.md:259]:
```suggestion
- If you are behind a proxy, set the following env variables
```

File: OLCNE/README.md [hunk 12]
```diff
@@ -27,20 +28,22 @@ will not work in a VirtualBox guest until nested virtualization support is
 released for Intel CPUs.
 
 ## Prerequisites
-1. Install [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-1. Install [Vagrant](https://vagrantup.com/)
+
+1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM
```
Comment (totalamateurhour) [inline OLCNE/README.md:32]:
```suggestion
1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up Vagrant with either VirtualBox or KVM
```

File: OracleRAC/README.md [hunk 13]
```diff
@@ -1,29 +1,18 @@
...
-2. Install [Vagrant](https://vagrantup.com/)
-3. Install extra packages such ruby-devel libvirt-devel
-  - `yum install -y ruby-devel libvirt-devel`
-4. Install vagrant-libvirt as user
-  - `vagrant plugin install vagrant-libvirt`
-5. You need to download Database binary separately
-
-#### Note: *Using KVM/libVirt provider you may need to disable or manage host firewall to permit NFS traffic with the guest VMs*
-#### Note: if you are going to use KVM on OL7/OL8 please read 'https://blogs.oracle.com/linux/getting-started-with-the-vagrant-libvirt-provider-for-oracle-linux'
+1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM
```
Comment (totalamateurhour) [inline OracleRAC/README.md:11]:
```suggestion
1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up Vagrant with either VirtualBox or KVM
```

**Comment by totalamateurhour** [comment]
@Djelibeybi 
> We only need a single `LICENSE` file in the root of the repo. All the rest can be removed.

And we keep THIRD_PARTY_LICENSES.txt as those can vary per directory?

**Comment by Djelibeybi** [comment]
> And we keep THIRD_PARTY_LICENSES.txt as those can vary per directory?

We only need one of these too. We can specify which components are covered in a single file. However to keep it simple, I'm happy to sort out the license files in a different PR tomorrow.

File: OLCNE/README.md [hunk 14]
```diff
@@ -1,5 +1,6 @@
-# Vagrantfile to setup Oracle Linux Cloud Native Environment on Oracle Linux 7
-This Vagrantfile will deploy and configure the following components:
+# Vagrant project to setup Oracle Linux Cloud Native Environment on Oracle Linux 7
```
Comment (totalamateurhour) [inline OLCNE/README.md:1]:
```suggestion
# Vagrant project to set up Oracle Linux Cloud Native Environment on Oracle Linux 7
```

**Comment by Djelibeybi** [comment]
Some files have been committed by mistake in the OracleLinux/7 directory. The .env.local and bof.sh files.

**Comment by AmedeeBulle** [comment]
> Some files have been committed by mistake in the OracleLinux/7 directory. The .env.local and bof.sh files.

Right -- the .gitignore is in the other PR :(

**Comment by Djelibeybi** [comment]
@AmedeeBulle I would suggest removing all the `LICENSE` changes from this PR (to simplify it further) and I'll do all the `LICENSE` consolidation in another PR later, once this is merged.

**Comment by AmedeeBulle** [comment]
@Djelibeybi I'll squash all the changes tomorrow to make it simpler and re-submit without the license changes

**Comment by AmedeeBulle** [comment]
@Djelibeybi please also consider moving the OLCNE `CONTRIBUTING.md` at the top level...

**Comment by Djelibeybi** [comment]
> @Djelibeybi please also consider moving the OLCNE `CONTRIBUTING.md` at the top level...

Good call, will do.

**Comment by AmedeeBulle** [comment]
Removed LICENSE changes and squashed all commits for the README upate.

This is now ready to be merged.

File: OracleDG/README.md (hunk 8 above)
Comment (rcitton) [reply OracleDG/README.md]: note as the prerequisites are different based on the provider

File: OracleFPP/README.md (hunk 9 above)
Comment (rcitton) [reply OracleFPP/README.md]: note as the prerequisites are different based on the provider

**Pull request merged**